psydac/api/essential_bc.py

```diff
@@ -7,23 +7,24 @@
 
 
 #==============================================================================
-def apply_essential_bc_1d_StencilVector(V, bc, a):
+def apply_essential_bc_1d_StencilVector(V, a, bc=None, axis=None, ext=None, order=0):
```

Comment:
What does a default value of `None` mean for the `bc` argument? I don't think the code can work with that value

Reply:
Now I see, you get the information from `bc` if provided, and otherwise you get it from `axis`, `ext` and `order`. Why do you want to give this flexibility to these low-level functions?